src/components/RecipeLinks.astro

```diff
@@ -0,0 +1,42 @@
+---
+import { getEntryBySlug } from 'astro:content';
+
+interface Props {
+	slugs: string[];
+}
+
+function convertSlug(slug: string) {
+	let corrected = slug;
+	if (slug.startsWith('/')) {
+		corrected = corrected.slice(1);
+	}
+	if (slug.endsWith('/')) {
+		corrected = corrected.slice(0, -1);
+	}
+	return corrected;
+}
+
+const slugs = Astro.props.slugs.map(convertSlug);
+
+const entries = await Promise.all(slugs.map((slug) => getEntryBySlug('docs', slug)));
```

Comment:
We could throw a custom error to make it clear what's wrong when you mistype a slug. The current error thrown is `Cannot read properties of undefined (reading 'slug')`
```suggestion
const entries = await Promise.all(slugs.map(async (slug, i) => {
  let entry = await getEntryBySlug('docs', slug)
  if (!entry) throw Error(`Could not find entry for slug "${Astro.props.slugs[i]}"`)
  return entry
}));
```
<br/>
<details>
<summary>Before</summary>
<br/>
<img src="https://user-images.githubusercontent.com/64310361/230600530-22cd545e-72dd-4e57-afa7-6b71eb1414fc.png" alt=""/>
</details>
<details>
<summary>After</summary>
<br/>
<img src="https://user-images.githubusercontent.com/64310361/230600715-cbe69131-c91d-4910-bed0-6c712fd03337.png" alt=""/>
</details>

Reply:
Thanks, @TheOtterlord!